src/EasyList.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,13 @@
+using Xunit;
+
+namespace EasyList.Tests
+{
+    public class UnitTest1
```

Comment:
This test should be removed as it's not being used.

Reply:
I am still learning unit testing, so it haven't implemented it here. This was just the initial stub.
I' ll post the testing code with right commit.

Reply:
@Anequit Done